netlify-functions-ecommerce/models.js

```diff
@@ -27,29 +34,47 @@ const orderSchema = new mongoose.Schema({
     type: String
   },
   paymentMethod: {
-    id: String,
-    brand: String,
-    last4: String
+    type: String,
+    get(v) {
+      return v == null ? v : JSON.parse(v);
+    },
+    set(v) {
+      if (v == null) {
+        return v;
+      }
+      return typeof v === 'string' ? v : JSON.stringify(v);
+    }
   }
-}, { optimisticConcurrency: true });
+}, { versionKey: false });
 
 const Order = mongoose.model('Order', orderSchema);
 
 module.exports.Order = Order;
 
 const cartSchema = new mongoose.Schema({
-  items: [{
-    _id: false,
-    productId: { type: mongoose.ObjectId, required: true, ref: 'Product' },
-    quantity: { type: Number, required: true }
-  }],
+  items: {
```

Comment:
Consider better ways to do this for API tables. Maybe just add a comment explaining why this is set for API tables?

Reply:
Fixed in 31a0907